README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+## hw01 - editing README.md
+
+-----
+
+# My name is Carolyn Hammond.
+
+![](https://66.media.tumblr.com/e1d62157f9298082303d249d1872f3a5/tumblr_mkqa55nS1n1snqyxgo1_1280.png)
+
+-----
+
+## School-Related Endeavors 
+
+### I am a **third year** at the University of Chicago double-majoring in **Public Policy** and **Linguistics**. This quarter in particular, I'm taking two *Chicago Studies* courses:
```

Comment:
Just for your future reference,  try not to make the body into headings to avoid confusion